chainer/training/extensions/log_report.py

```diff
@@ -91,7 +91,11 @@ def __call__(self, trainer):
                 fd, path = tempfile.mkstemp(prefix=log_name, dir=trainer.out)
                 with os.fdopen(fd, 'w') as f:
                     json.dump(self._log, f, indent=4)
-                os.rename(path, os.path.join(trainer.out, log_name))
+
+                log_path = os.path.join(trainer.out, log_name)
+                if os.path.exists(log_path):
+                    os.remove(log_path)
+                os.rename(path, log_path)
```

Comment:
I have tried on Windows. `shutil.move` worked well. Also, `os.path.exists` version worked well. In addition, `os.path.isfile` instead of `os.path.exists` also worked well.